content/overview/user-interface/views.md

```diff
@@ -6,14 +6,22 @@ weight: 10
 author: "people"
 ---
 
-The functionality in darktable is separated into six different views:
+The functionality in darktable is delivered through two main views and four subsidiary views:
+
+**Main Views**
+<br>Switch between main views by clicking the view name at the top right of the window:
 
 [lighttable](../../lighttable/_index.md)
 : Manage images and collections.
 
 [darkroom](../../darkroom/_index.md)
 : Develop a single image.
 
+The currently active view is highlighted.
+
+**Other Views**
```

Comment:
Let's keep the terminology consistent. Are they subsidiary views or "other" views? Given that the application uses the term "other" I'd go with that, but I agree it makes the text harder to write. On reflection I'm not sure it's worth the effort to call them "main" and "other/subsidiary". Perhaps it's better to just add a line before the map/tethering sections saying something like "the following views are available by clicking on the 'other' label"

Reply:
Agree with using "other" instead of subsidiary.

The main reason for splitting out the "Other" views is that the UI already casts them as different/lesser by relegating them to the dropdown.  So the manual should emphasise the ease of switching between lightable and darkroom which is where users will spend 90+% of their time.

Your call.

Reply:
On a practical note. I can make the corrections to the text in my fork alixtimix/dtdocs/patch-4 

Once I commit those changes, how do I send the updated material back to you?  A new push request? or what?

Reply:
If you update your branch that automatically updates the PR

Reply:
OK all done

Reply:
also note it isn't always called "other" -- when you change to the map view, then it is called "map", etc etc. So what it is called depends on what view you're currently in.